src/store/types.ts

```diff
@@ -1,22 +1,24 @@
 import Big from 'big.js'
 
-import { Buffer, BN } from '@c4tplatform/caminojs'
+import { Buffer, BN } from '@c4tplatform/caminojs/dist'
 import AvaAsset from '@/js/AvaAsset'
 import MnemonicWallet from '@/js/wallets/MnemonicWallet'
 import { ITransaction } from '@/components/wallet/transfer/types'
 import { AllKeyFileTypes, AllKeyFileDecryptedTypes } from '@/js/IKeystore'
 import { UTXO } from '@c4tplatform/caminojs/dist/apis/avm'
 import { UTXO as TxUTXO } from './modules/history/types'
 import { WalletNameType, WalletType } from '@/js/wallets/types'
+import { KeystoreFileKeyType } from '@/js/IKeystore'
 
 export interface RootState {
     isAuth: boolean
+    storedActiveWallet: null | WalletType
     activeWallet: null | WalletType
     wallets: WalletType[]
     address: String | null
     volatileWallets: WalletType[] // will be forgotten when tab is closed
     warnUpdateKeyfile: boolean
-    theme: 'night' | 'day'
+    walletsDeleted: boolean
```

Comment:
theme state is missing

Reply:
but this leads to this error in the suite app we need to define which of the theme is selected in the suite
<img width="314" alt="Screenshot 2023-03-22 at 14 54 33" src="https://user-images.githubusercontent.com/51858084/226944456-ab80f165-2c88-4183-9a22-1aab4b6e877c.png">

Reply:
@Chriserus this shouldn't be removed.

Reply:
Restored